src/components/flat-table/flat-table-row/flat-table-row.component.js

```diff
@@ -171,7 +172,6 @@ const FlatTableRow = React.forwardRef(
                     reportCellWidth:
                       index < rowHeaderIndex ? reportCellWidth : undefined,
                     leftPosition: leftPositions[index],
-                    pl: index === firstCellIndex() && expandable ? "4px" : null,
```

Comment:
I added this in v72.2.2 to fix the issues with `pl` not getting applied to the table-cell. Without this code, the arrow for `expandable` was out of line with the text in the flat-table-header. Does everything still line up in the `expandable` examples without this code?

Reply:
Yes, following the new specs for this task 👍🏻